src/Extensions/Activities/Internal/EventSource/ActivityEventSender.cs

```diff
@@ -116,6 +116,8 @@ private string SanitizeString(string value, string name, string activityName)
 			return value;
 		}
 
+		public void OnStop(Activity activity, object? payload = null) => throw new NotImplementedException();
```

Comment:
Suggest adding a message to this exception to make the code self-document.
Seems a bit odd to add the interface and then not implement it.

Reply:
Sorry this part committed by mistakes